Sources/Logto/Extensions/String.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  String.swift
+//
+//
+//  Created by Gao Sun on 2022/1/7.
+//
+
+import Foundation
+
+extension String {
+    static func fromUrlSafeBase64(string: String) -> String? {
+        guard let data = Data(base64Encoded: string
+            .replacingOccurrences(of: "-", with: "+")
+            .replacingOccurrences(of: "_", with: "/")
+            // Base64 encoding
+            .padding(toLength: string.count + string.count % 3, withPad: "=", startingAt: 0))
```

Comment:
just wondering do we need to align the padding "="  chars in js SDK as well, current `UrlSafeBase64` methods defined in silverhand essentials does not append those padding "=" chars

Reply:
actually this is a reverse action. we remove "="s on purpose when encoding to url-safe strings.